livelossplot/__init__.py

```diff
@@ -9,6 +9,12 @@
 from . import outputs
 from .version import __version__
 
+
+"""livelossplot
+
+Main repository: <https://github.com/stared/livelossplot>
```

Comment:
I have noticed out how to place a Readme.md in the documentation. 
You have to include:
```
.. include:: ../README.md
```
in this docstring.
I have checked it in my [test repository](https://github.com/Bartolo1024/test/blob/master/explib/__init__.py) and it [works](https://bartolo1024.github.io/test) :sunglasses: